tpcc/README.md

```diff
@@ -0,0 +1,56 @@
+# TPC-C Benchmark Using DBOS SDK
+
+This is a TPC-C TypeScript + Knex.js implementation written with [DBOS SDK](https://docs.dbos.dev/).
+
+## Running Locally
+
+First, start the database.
+DBOS workflow works with any Postgres database, but to make things easier, we've provided a nifty script that starts Postgres locally in a Docker container and creates a database:
+
+```bash
+export PGPASSWORD=dbos
+./start_postgres_docker.sh
+```
+
+Then, create some database tables.
+In this quickstart, we use [knex.js](https://knexjs.org/) to manage database migrations.
+Run our provided migration to create a database table:
+
+```bash
+npx knex migrate:latest
+```
+
+To load TPC-C data, we use [go-tpc](https://github.com/pingcap/go-tpc/tree/master).
+You can install it using the following script:
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/pingcap/go-tpc/master/install.sh | sh
```

Comment:
This line just hangs

Reply:
Also you have to `source ~/.bashrc` afterwards

Reply:
Solved.